src/chapter1/section2/Exercise14.java

```diff
@@ -49,7 +49,7 @@ public boolean equals(Object x) {
 		if (!this.who.equals(that.who)) {
 			return false;
 		}
-		if (this.when != that.when) {
+		if (this.when.equals(that.when)) {
```

Comment:
This should be `!this.when.equals(that.when)`

Reply:
Yes, sure.